Lab06Console/Program.cs

```diff
@@ -2,46 +2,81 @@
 
 namespace Lab06Console
 {
-    class Program
+  public class Program
+  {
+    static void Main(string[] args)
     {
-        static void Main()
-        {
-        //Console.WriteLine("This is the ZOO!");
-        Scorpions scorps = new Scorpions();
-        scorps.HairStyle = "Pineapple";
-        scorps.MemberCount = 5;
-        scorps.CountryofOrigin = "Germany";
-        Console.WriteLine("This is the ZOO!");
-      Metal[] metals = new Metal[]
+      Console.WriteLine("This is the ZOO!");
+      Metal scorps = new Scorpions();
+      scorps.HairStyle = "medium length and curly";
```

Comment:
Hairstyle, MemberCount, and CountryOfOrigin are things you may want to set in the Scorpions class (and for each of the concrete classes respectively) since a new instance of Scorpions would always want to have the same values for those characteristics.